storage/gcp/gcp.go

```diff
@@ -73,9 +72,17 @@ type objStore interface {
 
 // coord describes a type which knows how to sequence entries.
```

Comment:
Should `coord` be `sequencer` in the doc comment?

Reply:
d'oh, fixed